### ISSUE TITLE
open-editor: Unrecognised arguments "{}"

### ISSUE THREAD

**Issue by ersi-dnd** [The-Compiler/qutebrowser#826]
The editor setting is mysteriously not working.

**Comment by The-Compiler** [comment]
The command `:set general editor 'xterm -e nano "{}"'` which @Carpetsmoker posted in #41 works just fine for me as well.

It sounds like you actually bound a key to `:open-editor "{}"` instead of `:open-editor` - could this be the issue?

**Comment by ersi-dnd** [comment]
No. I know what says in the config file. I know what I typed in the address bar. I have been manually rewriting configs in tons of ways and it simply doesn't work. 

One of the things I tried was to install gvim, and then launch qutebrowser with clean profile. Even that didn't work. The setting simply does not work for me, no matter how I try.

Right now I tried with leafpad that I have here. When I type in the address bar, it briefly says 
 
 gvim -f "{}" Default value
 xterm -e nano "{}" Current value

And leafpad gives the same error. Everything always gives the same error.

**Comment by The-Compiler** [comment]
Can you please launch qutebrowser with `--temp-basedir`, then reproduce the issue and do `:report`? That should give me an idea of what's going on.

**Comment by ersi-dnd** [comment]
Hmm. But with --temp-basedir it works. I don't get it.

**Comment by ersi-dnd** [comment]
This was the first time I got it to work. I never got it to work by moving my profile away and starting a new profile...

**Comment by The-Compiler** [comment]
What do you mean with "moving my profile away and starting a new profile"? My guess is still that you have your key bound to something funny.

**Comment by ersi-dnd** [comment]
When I earlier said I launched with clean profile, I meant that I changed the name of my profile to qutebrowser.bak, and then launched the app. 

But now it's working. I think I traced the issue to the keys.conf. Now I have reconfigured everything as it was in my old profile and seems that nothing has gone wrong in the process and the external editor (nano of course) works. 

Really, earlier it didn't work for me even out of the box, and I tried for a month (more actually), but as soon as I brought it up, it began working. Thanks :)
